Documentation/api.md

```diff
@@ -243,6 +243,7 @@ The `NodeExporterCollectorNetClassConfig` resource works as an on/off switch for
 | -------- | ---- | ----------- |
 | enabled | bool | A Boolean flag that enables or disables the `netclass` collector. |
 | useNetlink | bool | A Boolean flag that activates the `netlink` implementation of the `netclass` collector. This implementation improves the performance of the `netclass` collector by omitting these metrics: `node_network_address_assign_type`, `node_network_name_assign_type`, `node_network_device_id`, `node_network_speed_bytes`. In addition, The `node_network_info` metric lacks the `duplex` label. |
+| ignoredNetworkDevices | *[]string | A list of network devices to be excluded from collection, using regular expressions. By default, the following list is used: `veth.*`, `[a-f0-9]{15}`, `enP.*`, `ovn-k8s-mp[0-9]*`, `br-ex`, `br-int`, `br-ext`, `br[0-9]*`, `tun[0-9]*`, `cali[a-f0-9]*` |
```

Comment:
Does it make sense to have 1 exclude list for each collector? I feel that in most (all?) cases, you want to apply the same list to both collectors.

Reply:
For this particular RFE it's fine to have only one, but in case later a customer wants to tune one and not the other, splitting a single setting into two would be a breaking change, or would need to keep both settings versions thus making it more complex, that's why I thought it would be better to have 2 right from the beginning. I'm fine with both options.

If we make a single option, where should it go? In `NodeExporterConfig`, in `NodeExporterCollectorConfig` or in a new struct for common network settings?

Reply:
I agree with @simonpasquier here, seems most likely user will want to ignore devices across collectors.
I'd propose to add this as a field to `NodeExporterConfig`. If we later need per collector ignore lists, they could be added and would override the list set in `NodeExporterConfig`.